File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLDoughnutChartProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLDoughnutChartProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String Title;
        public String TitleColor;
        public String TitleTextHeight;
        public String TitleFontString;
        public String InnerRadius;
        public String CurrentRadius;
        public String TotalValue;
        public String MarginSides;
        public String LabelColor;
        public String LabelHeight;
        public String LabelFontString;
        public String LegendWidth;
        public String LegendHeight;
        public String LegendFontString;
        public String AnimationCompleted;
        public String DeltaI;
        public String DeltaX;
        public String DeltaY;
        public String SliceClickFunction;
        public Object Tag;
        public String TabStopIndex;

        CCLDoughnutChartProps()
        {
            Data = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLDoughnutChartProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLDoughnutChartProps
    {
        public class DataItem
        {
            public string Label { get; set; }
            public float Value { get; set; }
            public string Color { get; set; }
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<DataItem> Data { get; set; }
        public string Title { get; set; }
        public string TitleColor { get; set; }
        public int TitleTextHeight { get; set; }
        public string TitleFontString { get; set; }
        public int InnerRadius { get; set; }
        public int CurrentRadius { get; set; }
        public int TotalValue { get; set; }
        public int MarginSides { get; set; }
        public string LabelColor { get; set; }
        public int LabelHeight { get; set; }
        public string LabelFontString { get; set; }
        public int LegendWidth { get; set; }
        public int LegendHeight { get; set; }
        public string LegendFontString { get; set; }
        public string AnimationCompleted { get; set; }
        public int DeltaI { get; set; }
        public int DeltaX { get; set; }
        public int DeltaY { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLDoughnutChartProps()
        {
            Data = new List<DataItem>();
        }
    }
}